expat/Makefile.am

```diff
@@ -95,16 +95,106 @@ EXTRA_DIST = \
     README.md \
     test-driver-wrapper.sh
 
+.PHONY: qa
+qa:
+	./qa.sh address
+	./qa.sh memory
+	./qa.sh undefined
+	./qa.sh coverage
+
+#page
+#### documentation
+
+dist_man_MANS = doc/xmlwf.1
+
+doc/$(am__dirstamp):
+	@$(MKDIR_P) doc
+	@: > doc/$(am__dirstamp)
+
+doc/xmlwf.1: doc/xmlwf.xml doc/$(am__dirstamp)
+if WITH_DOCBOOK
+	-rm -f $@
+	$(DOCBOOK_TO_MAN) $<
+	test -f MLWF.1 || mv XMLWF.1 $@
+else
+	@echo 'ERROR: Configure with --with-docbook for "make dist".' 1>&2
+	@false
+endif
+
+# https://www.gnu.org/software/automake/manual/automake.html#What-Gets-Cleaned
+.PHONY: clean-local
+clean-local: clean-local-check
+
+.PHONY: clean-local-check
+clean-local-check:
+	$(RM) doc/xmlwf.1 doc/$(am__dirstamp)
+
+EXTRA_DIST += \
+    doc/expat.png \
+    doc/reference.html \
+    doc/style.css \
+    doc/valid-xhtml10.png \
+    doc/xmlwf.xml
+
+#page
+#### library
+
+include_HEADERS = \
+    expat_config.h \
+    lib/expat.h \
+    lib/expat_external.h
+
+lib_LTLIBRARIES = libexpat.la
```

Comment:
No `lib/` prefix for `libexpat.la` any more?

Reply:
Will restore it.